### PR TITLE
Fix general spec version in README

### PR DESCRIPTION
**Pull request by oeway**
This PR fixes the general RDF URL in README.

@FynnBe Could you help with fixing the json schema file generation for the general RDF? The current one are named 0.3, see here in the gh-pages branch: https://github.com/bioimage-io/spec-bioimage-io/tree/gh-pages

### CODE REVIEW

**Comment by FynnBe** [comment]
but in the latest general RDF doc it's 0.2.0?
https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/rdf_spec_latest.md#bioimageio-resource-description-file-specification-020

**Comment by FynnBe** [comment]
ah the json.. sorry, just didn't read it correctly..

File: README.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ As a general guideline, please follow the model RDF spec to describe AI models a
 A BioImage.IO-compatible Resource Description File (RDF) is a YAML file with a set of specifically defined fields. 
 
 You can find detailed field definitions here: 
-   - [general RDF spec 0.3](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/rdf_spec_v0_3.md)
+   - [general RDF spec 0.2.0](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/rdf_spec_v0_2_0.md)
```
Comment (FynnBe) [inline README.md:23]: patch/micro version is actually ignored here on purpose... as it is setup currently only the latest patch version is really documented and has a json schema. there should not be significant/incompatible changes and if the specific patch/micro version is actually required we can go to the appropriate git tag.

Comment (FynnBe) [reply README.md:23]:
```suggestion
   - [general RDF spec 0.2.x](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/rdf_spec_v0_2.md)
```

File: README.md [hunk 2]
```diff
@@ -20,10 +20,10 @@ As a general guideline, please follow the model RDF spec to describe AI models a
...
 
 The specifications are also available as json schemas: 
-   - [general RDF spec 0.3 (json schema)](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/rdf_spec_v0_3.json)
+   - [general RDF spec 0.2.0 (json schema)](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/rdf_spec_v0_2_0.json)
```
Comment (FynnBe) [inline README.md:26]:
```suggestion
   - [general RDF spec 0.2.x (json schema)](https://github.com/bioimage-io/spec-bioimage-io/blob/gh-pages/rdf_spec_v0_2.json)
```

**Review by FynnBe — CHANGES_REQUESTED** [review]

**Comment by oeway** [comment]
@FynnBe Sounds good, thank you!

**Pull request merged**